sdk/cosmos/azure-cosmos/test/test_full_text_policy.py

```diff
@@ -0,0 +1,264 @@
+            "fullTextPaths": [
+                {
+                    "path": "/abstract",
+                    "language": "en-US"
+                }
+            ]
+        }
+        indexing_policy = {
+            "fullTextIndexes": [
+                {"path": "/abstract"}
+            ]
+        }
+        created_container = self.test_db.create_container(
+            id='full_text_container' + str(uuid.uuid4()),
+            partition_key=PartitionKey(path="/id"),
+            full_text_policy=full_text_policy,
+            indexing_policy=indexing_policy
+        )
+        properties = created_container.read()
+        assert properties["fullTextPolicy"] == full_text_policy
+        assert properties["indexingPolicy"]['fullTextIndexes'] == indexing_policy['fullTextIndexes']
+        self.test_db.delete_container(created_container.id)
+
+        # Create a container with a full text policy containing only default language
+        full_text_policy_no_paths = {
+            "defaultLanguage": "en-US"
+        }
+        created_container = self.test_db.create_container(
+            id='full_text_container' + str(uuid.uuid4()),
+            partition_key=PartitionKey(path="/id"),
+            full_text_policy=full_text_policy_no_paths,
+        )
+        properties = created_container.read()
+        assert properties["fullTextPolicy"] == full_text_policy_no_paths
+        self.test_db.delete_container(created_container.id)
+
+    def test_replace_full_text_container(self):
+        # Replace a container with a valid full text policy and full text indexing policy
+        full_text_policy = {
+            "defaultLanguage": "en-US",
+            "fullTextPaths": [
+                {
+                    "path": "/abstract",
+                    "language": "en-US"
+                }
+            ]
+        }
+        indexing_policy = {
+            "fullTextIndexes": [
+                {"path": "/abstract"}
+            ]
+        }
+        created_container = self.test_db.create_container(
+            id='full_text_container' + str(uuid.uuid4()),
+            partition_key=PartitionKey(path="/id"),
+            full_text_policy=full_text_policy,
+            indexing_policy=indexing_policy
+        )
+        properties = created_container.read()
+        assert properties["fullTextPolicy"] == full_text_policy
+        assert properties["indexingPolicy"]['fullTextIndexes'] == indexing_policy['fullTextIndexes']
+
+        # Replace the container with new policies
+        full_text_policy['fullTextPaths'][0]['path'] = "/new_path"
+        indexing_policy['fullTextIndexes'][0]['path'] = "/new_path"
+        replaced_container = self.test_db.create_container(
+            id=created_container.id,
+            partition_key=PartitionKey(path="/id"),
+            full_text_policy=full_text_policy,
+            indexing_policy=indexing_policy
+        )
+        properties = replaced_container.read()
```

Comment:
If container has been replaced, if we call the same test with original `created_container`, would we get the same result?

Reply:
nope - that's why the previous call to `container.read()` checks that the properties match those of the first container and not the second one. I'll add a check comparing the two against each other too.